zebra/zebra_rib.c

```diff
@@ -1970,8 +2019,9 @@ static void process_subq_route(struct listnode *lnode, uint8_t qindex)
 		char buf[SRCDEST2STR_BUFFER];
 
 		srcdest_rnode2str(rnode, buf, sizeof(buf));
-		zlog_debug("%u:%s: rn %p dequeued from sub-queue %u",
-			   zvrf ? zvrf_id(zvrf) : 0, buf, rnode, qindex);
+		zlog_debug("%s(%u):%s: rn %p dequeued from sub-queue %u",
+			   zvrf_name(zvrf), zvrf ? zvrf_id(zvrf) : 0, buf,
```

Comment:
maybe we can just replace VRF_UKNOWN to VRF_DEFAULT in zvrf_id and keep consistency everywhere?

Reply:
Changed.